backend/core/src/shared/email/email.service.ts

```diff
@@ -148,6 +148,60 @@ export class EmailService {
     )
   }
 
+  public async newlisting(listing: Listing, user: User) {
+    await this.loadTranslationsForUser(user)
+    if (this.configService.get<string>("NODE_ENV") == "production") {
+      Logger.log(
+        `Preparing to send a listing email to ${user.email} from ${this.configService.get<string>(
+          "EMAIL_FROM_ADDRESS"
+        )}...`
+      )
+    }
+
+    const rentRange = await this.getRentRange(listing)
+    await this.send(
+      user.email,
+      "New Listing",
+      this.template("new-listing")({
+        listing: listing,
+        rent: rentRange,
+      })
+    )
+  }
+
+  public async updateListingReminder(listing: Listing, user: User) {
+    await this.loadTranslationsForUser(user)
+    if (this.configService.get<string>("NODE_ENV") == "production") {
+      Logger.log(
+        `Preparing to send a reminder to update listing email to ${
+          user.email
+        } from ${this.configService.get<string>("EMAIL_FROM_ADDRESS")}...`
+      )
+    }
+
+    const rentRange = await this.getRentRange(listing)
+    await this.send(
+      user.email,
+      "Update Listing",
+      this.template("update-listing")({
+        listing: listing,
+        rent: rentRange,
+      })
+    )
+  }
+
+  // function to calculate rent - min of all mins and max of all maxs
+  private async getRentRange(listing: Listing) {
```

Comment:
I don't think this method needs to be async, since there aren't any `await`s in the method.